core/invoker/src/main/scala/whisk/core/containerpool/docker/DockerContainer.scala

```diff
@@ -60,6 +61,7 @@ object DockerContainer {
              network: String = "bridge",
              dnsServers: Seq[String] = Seq(),
              name: Option[String] = None,
+             useRunc: Boolean = true,
```

Comment:
WDYT about making `config` an implicit param to avoid method signature changes downstream ?

Reply:
i originally made it an implicit and @rabbah requested that it not be an implicit, so I reworked.

Reply:
an implicit boolean is suspect. I think @ddragosd was perhaps thinking about config as in whisk config? I prefer the explicit parameter here. Is there an implication to the other container factories of concerns?

Reply:
no implications to other container factories I believe. I just thought we can keep the PR smaller, but if it makes more sense to be explicit with this flag, 👍